src/NServiceBus.AwsLambda.SQS/SQSEventExtensions.cs

```diff
@@ -0,0 +1,95 @@
+﻿namespace NServiceBus.AwsLambda.SQS
+{
+    using System.Collections.Generic;
+    using System.IO;
+    using Amazon.SQS.Model;
+    using static Amazon.Lambda.SQSEvents.SQSEvent;
+
+    static class SQSEventExtensions
+    {
+        public static Message ToMessage(this SQSMessage sqsEventRecord)
+        {
+            var newMessage = new Message();
+
+            if (!string.IsNullOrEmpty(sqsEventRecord.MessageId))
```

Comment:
since we just assign them 1 to 1, do we need to have all the null checks? Assuming that the properties can be null by default on both types, assigning null to a property that would be null anyway, doesn't sound like something we need to prevent?